Sources/Swift/Helper/SentryCurrentDateProvider.swift

```diff
@@ -0,0 +1,20 @@
+@_implementationOnly import _SentryPrivate
```

Comment:
l: We dont need _SentryPrivate here
```suggestion
```

Reply:
Yes we do, we need it because of `getAbsoluteTime()`, which is in `SentryTime.h`